style.css

```diff
@@ -0,0 +1,61 @@
+.modal-dialog {
+  z-index: 9999;
```

Comment:
`z-index: 9999` is too large and is not a proper value. How about use 1024 or 2048? I know we will override it in every app. But the 9999 looks like a gold.